account_invoice_line_view_oaw/models/account_invoice_line.py

```diff
@@ -106,15 +77,17 @@ def _compute_base_amt(self):
             # set rate 1.0 if the transaction currency is the same as the base currency
             if inv_ln.currency_id == inv_ln.company_id.currency_id:
                 rate = 1.0
+            elif inv_ln.invoice_id.paid_date_currency_rate:
+                rate = inv_ln.invoice_id.paid_date_currency_rate
             else:
                 invoice_date = inv_ln.date_invoice or inv_ln.env.context.get(
                     "date", datetime.today().strftime("%Y-%m-%d")
```

Comment:
Better use context_today() instead?

Reply:
I guess you want to tag this part instead?
```
                invoice_date = inv_ln.date_invoice or inv_ln.env.context.get(
                    "date", datetime.today().strftime("%Y-%m-%d")
```
OK, I will create a follow PR on the your comments.